docs/source/demos/finnegans_wake_demo.rst

```diff
@@ -0,0 +1,55 @@
+Finnegan's Wake Demo
+====================
+
+.. figure:: https://cdn-images-1.medium.com/max/600/0*b42NPOnflrY07rEf.jpg
+    :width: 100%
+
+Overview
+--------
+
+.. important::
+
+    This demo requires connecting to a running network. By default the demo is hardcoded to connect to the local demo fleet.
+
+
+This demo is an example of a NuCypher decentralized network allowing Alice to share
+data with Bob using proxy re-encryption. This enables the private sharing of data between
+participants in public consensus networks, without revealing data keys to intermediary entities.
+
+
++------+-----------+----------------------------------------------------------------------------------------------+
+| Step | Character | Operation                                                                                    |
++======+===========+==============================================================================================+
+| 1    | Alice     | Alice sets a Policy on the NuCypher network (2/3) and grants access to Bob                   |
```

Comment:
Since we typically say `m-of-n` scheme, should we do the same here (?)
```suggestion
| 1    | Alice     | Alice sets a Policy on the NuCypher network (2-of-3) and grants access to Bob                   |
```